release-notes/9.0/preview/preview5/api-diff/Microsoft.AspNetCore.App/9.0-preview5_System.IO.Pipelines.md

```diff
@@ -0,0 +1,181 @@
+# System.IO.Pipelines
```

Comment:
I think this is confusing. It's making it look like we deleted a bunch of surface area and then later added a bunch of surface area, but really we just moved the library down to netcoreapp.  I suggest deleting this page and the corresponding one in netcoreapp.